app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -65,5 +55,16 @@ def textual_product_name
       name = @record.operating_system.try(:product_name)
       {:label => "Product Name", :value => name} if name
     end
+
+    def textual_guest_applications
+      num = @record.number_of(:guest_applications)
+      h = {:label => "Packages", :image => "guest_application", :value => num}
+      if num > 0
+        h[:title] = "Show the #{pluralize(num, "Package")} installed on this Container Image"
```

Comment:
add i18n support (the _ function). I think always writing 'Show the packages ...' is good enough.